app/components/agent/Index/installation.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Relay from 'react-relay';
+import { seconds } from 'metrick/duration';
+import shallowCompare from 'react-addons-shallow-compare';
+
+import Button from '../../shared/Button';
+import Dialog from '../../shared/Dialog';
+import Emojify from '../../shared/Emojify';
+import Panel from '../../shared/Panel';
+
+class AgentInstallation extends React.Component {
+  static propTypes = {
+    organization: React.PropTypes.shape({
+      agents: React.PropTypes.shape({
+        count: React.PropTypes.number.isRequired,
+        edges: React.PropTypes.array.isRequired
+      }).isRequired
+    }).isRequired,
+    relay: React.PropTypes.object.isRequired
+  };
+
+  state = {
+    isDialogOpen: false
+  };
+
+  componentDidMount() {
+    this._agentRefreshInterval = setInterval(this.fetchUpdatedData, 5::seconds);
```

Comment:
Maybe we should use `setTimeout` and then schedule another timeout once that's finished? Something like:

```
function refresh() {
  setTimeout(function() {
    this.props.relay.forceFetch(true, function() {
      this.refresh();
    })
  }, 5::seconds)
}
```

If requests are taking a long time (for some reason) they'll keep piling up, where as with this approach, it'll only schedule the next one after the first one finishes, if that makes sense?